doc/lsst.source.injection/reference/41_match_injected_outputs.rst

```diff
@@ -0,0 +1,201 @@
+.. _lsst.source.injection-ref-inspect:
+
+========================
+ Match Injected Outputs
+========================
+
+-----------------------
+ Process injected data
+-----------------------
+To create an injected outputs table,
+In ``drp_pipe`` there is a standard data processing pipeline for coadd-level injection.
+This pipeline is dynamically updated using ``make_injection_pipeline``.
+This example ``pipetask run`` command makes use of the ``injected_step3`` pipeline subset to run only tasks including and after ``inject_coadd`` within the typical ``step3``.
+This saves memory and runtime by skipping the expensive ``makeWarp`` and ``assembleCoadd`` tasks which usually occur before ``inject_coadd`` in the default ``step3``.
+
+.. code::
+
+    pipetask --long-log --log-file $LOGFILE \
+    run --register-dataset-types \
```

Comment:
Please never put `--register-dataset-types` into docs, not without a strong warning at least!